payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -65,11 +65,20 @@ internal class CardNumberController constructor(
     override val trailingIcon: Flow<TextFieldIcon?> = _fieldValue.map {
         val cardBrands = CardBrand.getCardBrands(it)
         if (accountRangeService.accountRange != null) {
-            TextFieldIcon(accountRangeService.accountRange!!.brand.icon, isIcon = false)
-        } else if (cardBrands.size == 1) {
-            TextFieldIcon(cardBrands.first().icon, isIcon = false)
+            TextFieldIcon.Trailing(accountRangeService.accountRange!!.brand.icon, isIcon = false)
         } else {
-            TextFieldIcon(CardBrand.Unknown.icon, isIcon = false)
+            val staticIcons = cardBrands.map { cardBrand ->
+                TextFieldIcon.Trailing(cardBrand.icon, isIcon = false)
```

Comment:
nit: and this is probably not really this review, but can we rename isIcon to isTintable